OpenSky_Get_Data.py

```diff
@@ -71,39 +61,36 @@ def getter(init_time, bounds, timer, anam):
                                   bounds=bounds,
                                   other_params=" and time-lastcontact<=15 ")
         flights.to_pickle(outf)
+        print("Wrote", outf)
```

Comment:
I'd prefer if this function doesn't print by default. Could you remove this, or maybe add a `verbose` parameter?

Reply:
Removed. So now it's back to printing only if a file already exists.